com.vrcfury.vrcfury/Editor/VF/Utils/AnimationClipExtensions.cs

```diff
@@ -116,6 +116,14 @@ public static class AnimationClipExtensions {
             clip.SetCurves(other.GetAllCurves());
         }
 
+        public static void CopyFromLast(this AnimationClip clip, AnimationClip other) {
+            EditorUtility.CopySerialized(other, clip);
```

Comment:
I don't believe CopySerialized should be here. The only property we really use this for is looping, but the new clip should never be looping anyways.